### PR TITLE
OKTA-587064 Update outdated logo requirements in OIN integration best practices

### PR DESCRIPTION
**Pull request by vanngo-okta**
## Description:
- **What's changed?** Update logo requirements in the outdated OIN integration best practices guide (this doc needs to be revamped in the very near future).
- **Is this PR related to a Monolith release?** No

### Resolves:

* [OKTA-587064](https://oktainc.atlassian.net/browse/OKTA-587064)

### CODE REVIEW

**Review by tovamarkowitz-okta — APPROVED** [review]
lgtm

**Review by tomaszwrobel-okta — APPROVED** [review]

File: packages/@okta/vuepress-site/docs/guides/oin-oidc-best-practices/main/index.md [hunk 1]
```diff
@@ -14,7 +14,7 @@ Due to the OIN restrictions of using an org authorization server and the authori
 
 A clear and well-designed logo helps customers find your app integration in the OIN and ensures that your brand is well represented. When you create your app submission in the OIN manager, be sure to upload a graphic that your customers will instantly recognize.
 
-The logo file must be smaller than 1 MB in size and be in either PNG, JPG, or GIF file format. For best results, use a PNG image with a transparent background, a landscape orientation, and use a minimum resolution of 420 x 120 pixels to prevent any unsightly upscaling.
+The logo file must be smaller than 1 MB and be in either PNG, JPG, or GIF file format. For best results, use a 200 x 200 pixels square-dimension PNG image with a transparent background. For more logo guidelines, see [Logo guidelines in OIN submission requirements](/docs/guides/submit-app-prereq/main/#logo-guidelines).
```
Comment (brentschaus-okta) [inline packages/@okta/vuepress-site/docs/guides/oin-oidc-best-practices/main/index.md:17]: Does the cross-reference need to be as long as it is?
```suggestion
The logo file must be smaller than 1 MB and be in either PNG, JPG, or GIF file format. For best results, use a 200 x 200 pixels square-dimension PNG image with a transparent background. See [Logo guidelines in OIN submission requirements](/docs/guides/submit-app-prereq/main/#logo-guidelines).
```

**Review by brentschaus-okta — APPROVED** [review]
LGTM

Just a small query/suggestion.

File: packages/@okta/vuepress-site/docs/guides/oin-oidc-best-practices/main/index.md (hunk 1 above)
Comment (vanngo-okta) [reply packages/@okta/vuepress-site/docs/guides/oin-oidc-best-practices/main/index.md:17]: Nope, it doesn't. Thanks!

**Comment by acrolinxdevdocs-okta** [comment]
## Acrolinx score **DRAFT 06.01.22**
 
A minimum Acrolinx Score of **40** is required. The total score is an average of the subscores.

Select **Total score** to review the Acrolinx scorecard for your article. Try to increase your individual scores, ex. Correctness. Your article will be clearer and more consistent with Microsoft standards.
 
| Article | **Total score**<br />**Required:70** | Word and phrases<br />(Brand, terms)<br />Preferred: 40 | Correctness<br />(Spelling, grammar)<br />Preferred: 40 | Clarity<br />(Readability)<br />Preferred: 10 | Inclusive language<br />(+ accesibility)<br />Preferred: 10 |
|---------|:------------------------------------:|:--------------------------------------------------------:|:--------------------------------------------------------:|:---------------------------------------------:|:------------------------------------------------------------:|
| [:white_check_mark:](# "Successfully processed") [packages/@okta/vuepress-site/docs/guides/oin-oidc-best-practices/main/index.md](https://github.com/okta/okta-developer-docs/blob/b0dd2adbf78b53f6ae1ff75350697aad0431ff72/packages%2F%40okta%2Fvuepress-site%2Fdocs%2Fguides%2Foin-oidc-best-practices%2Fmain%2Findex.md) | [68](https://okta.acrolinx.cloud/api/v1/checking/scorecards/f8dd32f9-b98a-4a2b-9cbf-6140d987ea57) | 45  | 80  | 41  | 100  |

  - The results from this response were procced by the [hugo-targets1.solution.acrolinx.sh](https://hugo-targets1.solution.acrolinx.sh) Acrolinx Cloud Host.
  - Scorecard links by default are only good for 7 days. [Link for this commit's Content Analysis Dashboard](https://okta.acrolinx.cloud/api/batch/gen.github.2ba921ed-ab30-4225-b3c9-ddea521c9f53?csBaseUrl=https%3A%2F%2Fokta.acrolinx.cloud&X-Acrolinx-Client-Locale=en)

**Pull request merged**